### ISSUE TITLE
Clear .gazebo/models cache for home-brew jobs

### ISSUE THREAD

**Issue by osrf-migration** [ignition-tooling/release-tools#27]
**Original report ([archived issue](https://osrf-migration.github.io/ignition-tooling-gh-pages/#!/osrf/release-tools/issues/27)) by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/)).**

----------------------------------------
I've been looking at persistent test failures recently, specifically [gzTest.SDF](http://build.osrfoundation.org/view/homebrew/job/gazebo-any-devel-homebrew_bcn-amd64/101/testReport/(root)/gzTest/SDF/). It's a test that expects `gzsdf print` of a world file to match a shasum. That test passes on my local machine. I ran a test build with [additional debugging print statements](http://build.osrfoundation.org/job/gazebo-any-devel-homebrew-amd64/41/), and I noticed two lines that differ in the definition of the sun model.

The short version is that the home-brew jenkins machines have an old cache of gazebo models and it's causing this test to fail. I just cleared the `~jenkins/.gazebo/models` folder on t1000 and started a new build, which should fix the failing test.

We should clear the `~/.gazebo/models` folder in the home-brew jenkins scripts to keep the cache from getting outdated.
